Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/config/PortalMenuConfig.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * #%L
+ * This file is part of "Apromore Core".
+ * %%
+ * Copyright (C) 2018 - 2021 Apromore Pty Ltd.
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Lesser Public License for more details.
+ *
+ * You should have received a copy of the GNU General Lesser Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/lgpl-3.0.html>.
+ * #L%
+ */
+package org.apromore.portal.config;
+
+import org.apromore.plugin.portal.PortalLoggerFactory;
+import org.slf4j.Logger;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.core.io.Resource;
+import org.json.JSONObject;
+import org.json.JSONArray;
+
+import org.slf4j.Logger;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import lombok.Getter;
+
+import org.apromore.plugin.portal.PortalLoggerFactory;
+
+@Component("menuConfig")
+public class PortalMenuConfig {
+
+	private static final Logger LOGGER = PortalLoggerFactory.getLogger(PortalMenuConfig.class);
```

Comment:
This PortalMenuConfig needs a unit test with different types of menu available, test the exception handling as well.